### PR TITLE
Added Face Mask Detection model and updated steps to include MaskDete…

### PR DESCRIPTION
**Pull request by NosBos**
…ction step

### CODE REVIEW

File: face_mask_detection/face_detector.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+            crop_img = np.expand_dims(crop_img, axis=0)
+
+            # get the prediction from the model.
+            prediction = self.model.predict(crop_img)
+
+            index = np.argmax(prediction)
+            res = categories[index]
+
+            if index == 0:
+                color = (0, 0, 255)
```
Comment (DavidHaighton) [inline face_mask_detection/face_detector.py:61]: Change the colours around

File: face_mask_detection/model/data_collection.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+count = 0
+
+class_path = 'mask'
+while True:
+    _, frame = cap.read()
+    img = frame.copy()
+    try:
+        # Make detection
+        x, y, w, h = get_detection(frame)
+        cv2.rectangle(frame, (x, y), (x+w, y+h), (0, 0, 255), 2)
```
Comment (DavidHaighton) [inline face_mask_detection/model/data_collection.py:49]: change it around here too maybe?

File: face_mask_detection/face_detector.py [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+            else:
+                color = (0, 255, 0)
```
Comment (Justgo13) [inline face_mask_detection/face_detector.py]: I think these colors should be reverted

**Review by DavidHaighton — APPROVED** [review]

File: face_mask_detection/face_detector.py (hunk 3 above)
Comment (NosBos) [reply face_mask_detection/face_detector.py]: Updated index

**Review by Justgo13 — APPROVED** [review]

**Pull request merged**